app/main.py

```diff
@@ -1,3 +1,70 @@
+from __future__ import annotations
+from typing import Any
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self,) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self,) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Any) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        else:
+            return Distance(self.km + other)
+
+    def __iadd__(self, other: Any) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        else:
+            self.km += other
+            return self
+
+    def __mul__(self, other: Any) -> Distance:
+        if isinstance(other, Distance):
+            pass
+        else:
+            return Distance(self.km * other)
```

Comment:
```suggestion
        return Distance(self.km * other)
```
Check is redundant
Fix the same into truediv